app/helpers/notifications_helper.rb

```diff
@@ -23,4 +23,12 @@ def any_active_filters?
   def filtered_params(override = {})
     filters.merge(override)
   end
+
+  def archive_selected_button(custom_class=nil)
+    action = params[:archive] ? 'unarchive' : 'archive'
+    button_tag(type: "button",
+               class: "archive_toggle #{action}_selected #{custom_class}") do
+      "#{action} selected".capitalize
```

Comment:
@karpiu Good call out! From what I have read, html is 'mostly' case-sensitive. I gave my suggested change a try, and it breaks things. Sorry about that!